defaults/main.yml

```diff
@@ -45,12 +53,13 @@ django_stack_es_host_url: http://localhost:9200
 django_stack_es_ca_path: /etc/ssl/certs/testca_freedom_press.pem
 
 #npm settings
-django_stack_npm_install_cmd: npm install
+django_stack_npm_install_cmd: "npm install; npm run start"
```

Comment:
i honestly dont think it matters -- ive had a script bomb out before here and it just kills the entire container and the play stops with an error. im hesitant to change it though -- its running in `ash` , not `bash` under a alpine linux container